content/md/en/docs/tutorials/work-with-pallets/use-macros-in-a-custom-pallet.md

```diff
@@ -372,8 +359,9 @@ To implement this logic in the proof-of-existence pallet:
    cargo check -p node-template-runtime
    ```
 
-You can refer to the node template [solution](https://github.com/substrate-developer-hub/substrate-node-template/tree/tutorials/solutions/proof-of-existence) if you get stuck.
-You can also check the [commit diff](https://github.com/substrate-developer-hub/substrate-node-template/commit/773d72752f3598e5d405b48c6f716f4153c95070#diff-f191d0eb0afa874a64fb9be550a275f957e220b9076c87d4085c2797ca4e310c) to see the exact changes from the base template.
+> **_Something not working?_**
```

Comment:
```suggestion
```
Please move these solution links to the end. Including it here implies that you *expect* it not to work. That means it is a bad tutorial. It's find to provide a solution reference, but we've just had them verify that it compiles, so this seems weird.